ui/raidboss/data/05-shb/alliance/the_copied_factory.js

```diff
@@ -500,6 +540,283 @@
       delaySeconds: 5,
       infoText: {
         en: 'Get in the bubble',
+        de: 'Geh in die Kuppel',
+      },
+    },
+  ],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'replaceSync': {
+        '9S-Operated Walking Fortress': '9S\' mehrbeiniger Panzer',
```

Comment:
Can you add syncs for 9S-Operated Flight Unit? I think the `Flight Unit` translation is conflicting with this.

Reply:
Yeah you are right. I will have a look if i can avoid these in the future.